SIGNALESP/SIGNALESP.ino

```diff
@@ -247,16 +248,41 @@ void loop() {
 		if (!command_available) { cmdstring = ""; }
 		blinkLED = true;
 	}
+	
 	if (fifousage < FiFo.count())
 	  fifousage = FiFo.count();
-
-	while (FiFo.count()>0) { //Puffer auslesen und an Dekoder uebergeben
-		aktVal = FiFo.dequeue();
-		state = musterDec.decode(&aktVal);
-		if (state) blinkLED = true; //LED blinken, wenn Meldung dekodiert
-		if (FiFo.count()<120) yield();
+  while (FiFo.count()>0) { //Puffer auslesen und an Dekoder uebergeben
+  		aktVal = FiFo.dequeue();
+	  	state = musterDec.decode(&aktVal);
+		  if (state) blinkLED = true; //LED blinken, wenn Meldung dekodiert
+//      if (FiFo.count()<120) yield();
 	}
-
+  
+#ifdef _DEBUG_DEV_SERIAL
+  if (Serial.available()) {
+    unsigned char c = Serial.read();
+//    if (c != char(10))
+//      Serial.println(".");
+    switch(c) {
+      case 'c':
+        Serial.println("marc: 0x" + String(cc1101::currentMode(), HEX));
+        Serial.println("fifo: " + String(FiFo.count()) + ", max. " + String(fifousage));
+#ifdef _CC1101_DEBUG_CONFIG
+        cc1101::dumpConfigRegister();
+#endif
+        break;
+      case 'd':
+        dumpEEPROM();
+        break;
+      case 's':
+        MSG_PRINTLN("Test Test Test");
```

Comment:
Hiermit wird per ethernet ausgegeben, ist das so gewollt?

Reply:
ja (teste Verbindung zu FHEM). Ist eh in dem Abschnitt für Eingaben an der seriellen Schnittstelle.

Reply:
Okay verstanden